experimentals/react-native-uifabric/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "react-native-uifabric",
+  "version": "0.12.10",
+  "description": "A react-native companion to fabric web that only depends on stock controls and theming",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "private": true,
+  "scripts": {
+    "build": "just-scripts build --useMetro",
+    "just": "just-scripts",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "lint": "just-scripts eslint",
+    "start": "just-scripts dev",
+    "start-test": "just-scripts jest-watch",
+    "update-snapshots": "just-scripts jest -u",
+    "verify-api": "just-scripts verify-api-extractor",
+    "update-api": "just-scripts update-api-extractor",
+    "run-win32": "rex-win32 --bundle RNTester.win32 --component RNTesterApp --basePath ./dist/win32/dev",
+    "run-win32-web": "rex-win32 --bundle RNTester.win32 --component RNTesterApp --basePath ./dist/win32/dev --useWebDebugger"
+  },
+  "dependencies": {
+    "@uifabricshared/foundation-composable": "0.5.1",
+    "@uifabricshared/foundation-compose": "1.5.0",
+    "@uifabricshared/foundation-settings": "0.4.1",
+    "@uifabricshared/foundation-tokens": "0.4.0",
+    "@uifabricshared/immutable-merge": "0.2.0",
+    "@uifabricshared/theming-ramp": "0.5.0",
+    "@uifabricshared/theming-react-native": "0.3.0",
+    "tslib": "^1.9.3"
+  },
+  "devDependencies": {
+    "@office-iss/react-native-win32": "0.26.2",
+    "@types/dedent": "0.7.0",
+    "@types/enzyme": "^2.8.11",
+    "@types/es6-collections": "^0.5.29",
+    "@types/es6-promise": "0.0.32",
+    "@types/jest": "^24",
+    "@types/node": "^12.11.2",
+    "@types/prop-types": "15.5.1",
+    "@types/react": "16.9.11",
+    "@types/react-dom": "16.0.7",
+    "@types/react-native": "0.60.19",
+    "@types/react-test-renderer": "16.9.0",
+    "@types/webpack-env": "1.13.6",
+    "@uifabricshared/build-native": "^0.1.1",
+    "@uifabricshared/eslint-config-rules": "^0.1.1",
+    "dedent": "*",
+    "enzyme": "^3.1.0",
+    "enzyme-adapter-react-16": "^1.0.0",
+    "enzyme-to-json": "^3.1.1",
+    "office-ui-fabric-react": "^7.28.2",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "react-native": "https://github.com/microsoft/react-native/archive/v0.60.0-microsoft.35.tar.gz",
+    "react-native-windows": "0.60.0-vnext.115",
+    "react-test-renderer": "16.8.6"
+  },
+  "peerDependencies": {
+    "react": "16.8.6",
```

Comment:
uifabric shouldn't have such specific peer dependencies.  -- We should really start doing semver here.
We should be ok with any 0.60 release of react-native.

Reply:
I opened Issue #85 to track this as I think this needs to be done more broadly.  For this particular case will "react-native": "^0.60" resolve correctly against the tarball we are referencing for react-native-windows?

Reply:
I'm just copying this thing over from `sdx-platform`.

Reply:
I'll change to ^16.0.0